### PR TITLE
Support python 2

### PR DESCRIPTION
**Pull request by simba3447**
This pull request is related to the issue [#5](https://github.com/ankitjain28may/getit/issues/5).
I changed that urlopen is imported from urllib2 when python version is 2.

### CODE REVIEW

File: getit/getit.py [hunk 1]
```diff
@@ -5,7 +5,11 @@
...
 from time import time
-from urllib.request import urlopen
+from contextlib import closing
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
 
 from colorama import Fore, Style, init
 init()
```
Comment (ankitjain28may) [inline getit/getit.py:18]: check line 249, You will find `input()` syntax which is available only in python3

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

**Review by ankitjain28may — CHANGES_REQUESTED** [review]
It shows this error
```
  File "getit.py", line 123
SyntaxError: Non-ASCII character '\xe2' in file getit.py on line 124, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

**Comment by simba3447** [comment]
I don't see that error, but I added encoding declaration. Can you check if the error occurs again?

**Review by ankitjain28may — APPROVED** [review]
LGTM

**Pull request merged**